android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -241,6 +255,24 @@ private TokenDetails readTokenDetails(Map<String, Object> jsonMap) {
         return o;
     }
 
+    private Message decodeChannelMessage(Map<String, Object> jsonMap){
+        if(jsonMap==null) return null;
```

Comment:
Again, spacing after `if`, please. i.e.:

```suggestion
        if (jsonMap==null) return null;
```

Reply:
Same as https://github.com/ably/ably-flutter/pull/23#discussion_r471921091

052467d338e6fd8a37e3474c03b355e0ec1b8fdc